packages/terra-application-utility/src/ApplicationUtility.js

```diff
@@ -0,0 +1,12 @@
+import ApplicationHeaderUtility from './ApplicationHeaderUtility';
+import ApplicationMenuUtility from './ApplicationMenuUtility';
+import UtilityUtils from './_Utils';
+
+const ApplicationUtility = {
+  ApplicationHeaderUtility,
+  ApplicationMenuUtility,
+  UtilityUtils,
```

Comment:
Naming is hard

Reply:
Does utils need to be exported here?

Reply:
Yeah, the passed in menu item config should use the location constants stored in `UtilityUtils` for the `contentLocation` attribute.